setup.py

```diff
@@ -78,7 +85,8 @@ def read(*paths):
     ],
 
     python_requires='>=3.6.*',
-    install_requires=['pyvista>=0.32.0',
+    install_requires=['cython',
```

Comment:
Really, we should be using `cython` within a `pyproject.toml` build requires section.

Reply:
Agreed, I can drop this commit from the PR if you're updating to a `pyproject.toml`

Reply:
Dropped.